src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1742,6 +1742,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         ERR_ValueTupleResolutionAmbiguous3 = 37305
 
+        ERR_CommentsAfterLineContinuationNotAvailable1 = 37306
```

Comment:
This error shouldn't be needed now, as it handle by the `Parser.CheckFeatureAvailability` code.